src/patternfly/components/DescriptionList/description-list.scss

```diff
@@ -18,16 +18,21 @@ $pf-c-description-list--breakpoint-map: build-breakpoint-map("base", "md", "lg",
   --pf-c-description-list__term--FontSize: var(--pf-global--FontSize--sm);
 
   // Horizontal
-  --pf-c-description-list--m-horizontal__term--width: 14ch;
+  --pf-c-description-list--m-horizontal__term--width: 12ch;
   --pf-c-description-list--m-horizontal__description--width: minmax(10ch, auto);
-  --pf-c-description-list--m-horizontal__group--GridTemplateColumns: var(--pf-c-description-list--m-horizontal__term--width) var(--pf-c-description-list--m-horizontal__description--width);
+
+  // use --pf-c-description-list__term--width here as it is re-defined on line 45
```

Comment:
since this comment only applies to the declaration and not a block, it can be on the same line/after the declaration and you can maintain the spacing here.

Reply:
Opps!